src/style.css

```diff
@@ -99,3 +250,10 @@ footer p {
   text-decoration: none;
   cursor: pointer;
 }
+
+.name {
+  font-size: 20px;
+  font-weight: 600;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  margin: 6px;
+}
```

Comment:
- [ OPTIONAL] There are similar CSS rule name which name "name" it's can be created different name 

![image](https://user-images.githubusercontent.com/24355301/175649825-65c4cd30-d2a8-4900-8ad1-86378473b916.png)